### ISSUE TITLE
Upload Sources to Modrinth/Maven

### ISSUE THREAD

**Issue by sylv256** [jellysquid3/Sodium#2651]
### Request Description

Presently, I'm using Sodium from the Modrinth Maven. This comes with the unfortunate downside that I do not have access to a sources `.jar` which would be useful during development.

**Comment by jellysquid3** [comment]
Yeah. We need to host a Maven for other reasons too (so that Fabric API dependencies are declared in the Maven POM.) We had one for a short while but it wasn't maintained.
